### ISSUE TITLE
ModuleNotFoundError for latest Minio version  (>6.0) 

### ISSUE THREAD

**Issue by patvdleer** [theriverman/django-minio-backend#15]
```
File "/home/patrick/PycharmProjects/fpo-api/fpo/core/models/game.py", line 8, in <module>
    from django_minio_backend import MinioBackend
  File "/home/patrick/PycharmProjects/fpo-api/venv/lib/python3.7/site-packages/django_minio_backend/__init__.py", line 2, in <module>
    from .models import *
  File "/home/patrick/PycharmProjects/fpo-api/venv/lib/python3.7/site-packages/django_minio_backend/models.py", line 12, in <module>
    import minio.definitions
ModuleNotFoundError: No module named 'minio.definitions'
```

**Comment by patvdleer** [comment]
requirements.txt
```
minio~=6.0.0,<7
```

**Comment by ErrorInPersona** [comment]
Yup, can confirm this. Crashed our entire project :/

**Comment by ErrorInPersona** [comment]
In case someone is having a similar bad day...:
I hat do rebuild my containers. I pinned 2.2.0 in my requirements (worker fine). As you yanked that version, pip took the latest version. However not even downgrading to 2.0.0 fixed the issue.
Even changing the the fields in my models did not do enough, as django_minio_backend is imported in the migration files as well. So I had to delete them as well, to the project up and running. I guess a more exact version pinning with minio could help.
